src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$card-width: 200px;
+$media-requirements:
+  488px 2,
+  768px 3,
+  1024px 4;
 
-  max-width: 944px;
+.catalog {
   margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+
+  grid-template-columns: repeat(1, $card-width);
+
+  @each $min-screan, $columns in $media-requirements {
+    @media (min-width: $min-screan) {
+      grid-template-columns: repeat($columns, $card-width);
+    }
+  }
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
     height: 300px;
+    
```

Comment:
this is empty line also not necessary, because all styles refer  'Display & Box Model'